contracts/TransparentForwarder.sol

```diff
@@ -41,7 +43,8 @@ contract TransparentForwarder is AccessControlEnumerable {
         bool isWhitelisted = staking.isWhitelisted{value: msg.value}(
             msg.sender
         );
-        require(isWhitelisted, "Not whitelisted");
```

Comment:
can revert this change, since we should avoid having to redeploy transparent forwarder for existing testnets @SamAg19